types/jspdf/jspdf-tests.ts

```diff
@@ -174,9 +176,9 @@ function test_rect_squares() {
     doc.setDrawColor(0);
     doc.setFillColor(255, 0, 0);
     doc.rect(120, 20, 10, 10, 'FD'); // filled red square with black borders
-    doc.setDrawColor(0);
-    doc.setFillColor(255, 255, 255);
```

Comment:
Why this case was removed? I think this is a valid case with three numbers. Please explain.

Reply:
It's still there 3 lines above.